README.md

```diff
@@ -51,9 +51,43 @@ optional arguments:
                         If given, canNotUseMemoryManagementFunctions=false
 ```
 
-##### Example: 
+### How do I build an FMU from python code with third-party dependencies?
+
+Often, Python scripts depends on non-builtin libraries like `numpy`, `scipy`,... . _PythonFMU_ does not package a full environment within the FMU. However you can package a `requirements.txt` or `environment.yml` file within your FMU. And using `pythonfmu deploy`, your end user will be able to update is local Python environment. The steps to achieve that:
```

Comment:
I'd move this entire paragraph to line 65, replacing the `For the FMU end user` item